contracts/Zapper.sol

```diff
@@ -66,18 +66,24 @@ contract Zapper is AccessController, ReentrancyGuardUpgradeable, UUPSUpgradeable
         uint256 collateralInBaseStableAmount;
         uint256 archAmount;
 
-        // Check if we need are using existing arch tokens owned by user or buying new ones
+        // Check if we are using existing arch tokens owned by user or buying new ones
         if (useUserArch == true) {
             // We are using owners arch tokens, transfer from msg.sender to address(this)
             collateralInBaseStableAmount = stableCoinAmount;
             archAmount = _transferUserArchForPosition(stableCoinAmount, cycles, maxSlippageAllowed, addressBaseStable);
         } else {
-            // Need to buy Arch tokens. We need to split the stable amount between what we'll as collateral what we'll use to buy Arch
+            // Need to buy Arch tokens. We need to split the stable amount between what we'll use as collateral and what we'll use to buy Arch
             uint256 coinsToPayForArchAmount;
             (collateralInBaseStableAmount, coinsToPayForArchAmount) = _splitStableCoinAmount(stableCoinAmount, cycles, path, addressBaseStable);
-            // By arch tokens. Dont enforce min as we dont quite know what the minimum is. If we dont have enough this will fail when we try to use arch
+            // Buy arch tokens. Dont enforce min as we dont quite know what the minimum is. If we dont have enough this will fail when we try to use arch
             // to open position.
-            _uniswapRouter.swapExactTokensForTokens(coinsToPayForArchAmount, 0, path, address(this), block.timestamp + 2 minutes);
+            uint256[] memory amounts = _uniswapRouter.swapExactTokensForTokens(
```

Comment:
Are you using the amounts? Why get them and save them?

Reply:
I have no idea why this is here, I don't remember adding this. I can change it back if you don't remember doing this either, it might have been pulled in with the previous commits I built on top of.